.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Pipeline
+on: [push]
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.5, 2.6]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler: 1.17.3
+        bundler-cache: true
+    - name: Unit Tests
+      run: bundle exec rspec
```

Comment:
The Jenkinsfile was using rake rather than rspec here, but his is fine as long as you double checked that there was nothing special being added by the rake task.

Reply:
Double checked that the [rake task](https://github.com/Invoca/invoca-utils/blob/master/Rakefile) simply runs rspec with the default test pattern. 👍  I believe I can also take credit for removing the `shoulda` gem last minute in the rspec conversion and forgetting to run a `bundle install` 🙃